README.md

```diff
@@ -22,28 +22,25 @@
 **Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
 
 
-To get a local copy up and running follow these simple example steps.
-
+Clone the repository to your machine
```

Comment:
- [ ] This section must present instructions on setting up your project locally. Please focus on the steps involved in cloning the repository instead of the linter setup. That information must be the guideline for someone interested in checking your app running on their machines. For more informations click [here](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository).

Example:

<img width="614" alt="Screen Shot 2022-08-18 at 10 31 13 AM" src="https://user-images.githubusercontent.com/47673311/185407888-4dfde1df-835d-49aa-8d3a-0fa8af5979c9.png">